setup.py

```diff
@@ -42,7 +42,7 @@ def get_version():
         'Topic :: Internet :: WWW/HTTP :: Dynamic Content',
     ],
     install_requires=[
-        'Django>=1.6',
+        'Django>=1.6,<1.8',
```

Comment:
Could you leave this how it was? It actually makes it more complicated for when we upgrade. TravisCI has the appropriate versions pinned for testing https://github.com/ambitioninc/django-localized-recurrence/blob/develop/.travis.yml#L10-L11